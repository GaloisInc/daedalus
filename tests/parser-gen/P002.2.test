P002.ddl
--gen
--interp=inputs/P002.2.inp
