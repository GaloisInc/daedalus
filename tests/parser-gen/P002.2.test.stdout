context:
  "Main"
input: File context:
Length: 2 (0x2) bytes
0000:   30 [1;91m31[0m                                     0[1;91m1[0m

